src/components/UiFormInputs/UiInputText/UiInputText.vue

```diff
@@ -36,6 +36,8 @@
                     :icon="icon"
                     :minlength="min"
                     :maxlength="max"
+                    :min="minsize"
```

Comment:
minSize should be camel case ?

Reply:
If you copy the UiInputText from Firm24 it already have all that:

https://github.com/legalthings/firm24/blob/master/src/components/Global/UiFormInputs/UiInputText/UiInputText.vue

Reply:
Fixed